python/triton/compiler/compiler.py

```diff
@@ -236,6 +236,15 @@ def filter_traceback(e: BaseException):
         e.__traceback__ = frames[0]
 
 
+def triton_spirv_dump(data, file_name, ext):
+    spv_path = os.getenv("TRITON_XPU_DUMP_SPIRV_KERNEL_ARGS")
+    if not os.path.exists(spv_path):
+        os.makedirs(spv_path)
+    spv_name = f"{spv_path}/{file_name}.{ext}"
```

Comment:
```suggestion
    spv_name = os.path.join(spv_path, f"{file_name}.{ext}")
```

`os.path.join` is better